### ISSUE TITLE
Switch from zip files to .gdmod

### ISSUE THREAD

**Issue by otDan** [GodotModding/godot-mod-loader#22]

**Comment by ithinkandicode** [comment]
I don't think we need to worry about people unzipping mods. Generally I think it's a good thing to keep in mind when offering support, but I think it's also easy to sacrifice ease of use to cover edge cases of people misunderstanding things. And if we're gonna use Thunderstore then people manually downloading ZIPs won't be an issue anyway.

I'm also hesitant to rename a file extension in an arbitrary way, as it can obfuscate the file's purpose/function. Whether it's a ZIP or a PCK, I think the file extension should match the file format.

Still, we should resolve this ZIP vs. PCK discussion, so I've raised a new issue for it: #24

**Comment by ithinkandicode** [comment]
@otDan Are we ok to close this? I think we landed on using ZIPs, partly because they can be read by Thunderstore without needing extra bits

**Comment by Qubus0** [comment]
@ithinkandicode do we still want to rename the ending though? it would basically be a manual operation to change .zip to .gdmod and back. one extra step during development (might make automating that through pipelines  harder?) but less confusion for (manual) installation

**Comment by KANAjetzt** [comment]
I think Thunderstore only operates with plain .zip files

**Comment by Qubus0** [comment]
welp, then we can't change that.

**Comment by otDan** [comment]
I will close this for now, we can probably discuss this later on if really needed

**Comment by CST1229** [comment]
> I think Thunderstore only operates with plain .zip files

not everyone uses Thunderstore though. for other games, installing files manually can be the *main* method for getting mods.
